### PR TITLE
[WIP]: Full Audit on Linux Security Best Practices

### LINKED ISSUE #1855: Full Audit: Linux server security best practices
Due to the Product bulletin from 15 February 2017, this article may need a full audit. The bulletin offers a few POCs.

https://github.com/rackerlabs/rackspace-how-to/blob/master/content/cloud-servers/linux-server-security-best-practices.md

- [x] SME review
- [ ] Editing review
- [ ] Quality Checklist
- [ ] Audit date update

Ref issue #1850

### PR DESCRIPTION
**Pull request by nimbinatus**
Fixes #1855

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5ed473cad5/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits are attached. Please let me know if you have any questions. Thanks!

[Linux server security best practices_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/911436/Linux.server.security.best.practices_editsKH.pdf)
[Linux server security best practices_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/911437/Linux.server.security.best.practices_editsKH.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-37c807a69b/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-37c807a69b/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-358f56404e/how-to/).

File: content/cloud-servers/linux-server-security-best-practices.md [hunk 1]
```diff
@@ -280,81 +395,106 @@ You can save the following template/example iptables ruleset in the appropriate
...
+time that will cause the least amount of disruption during the (normally very short) downtime while your server is rebooting.
 
-To check for and install updates on Ubuntu and Debian systems, run:
+To check for and install updates on Ubuntu and Debian systems, run the following commands:
 
     sudo apt-get update
     sudo apt-get upgrade
 
-To check for and install updates on CentOS, Red Hat, and Fedora systems, run:
+To check for and install updates on CentOS, Red Hat, and Fedora systems, run the following commands:
```
Comment (stephamon) [inline content/cloud-servers/linux-server-security-best-practices.md]: There's only one command following this line, so "run the following command"

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-512f62f4da/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**